dll/ntdll/rtl/libsupp.c

```diff
@@ -917,7 +917,6 @@ RtlComputeImportTableHash(IN HANDLE FileHandle,
                           OUT PCHAR Hash,
                           IN ULONG ImportTableHashSize)
 {
-    UNIMPLEMENTED;
```

Comment:
This is basically `DPRINT1("RtlComputeImportTableHash is UNIMPLEMENTED\n");`
This will make it obvious to the user via a log message that an unimplemented function is called. That different from returning a value to the caller.
I'd say this is valuable, unless it causes debug spam in an application that calls it a lot.

Reply:
@ThFabba Oh, that makes sense of having `UNIMPLEMENTED` in the code then. I blame my ignorance for that. :disappointed:

Guess I have shall to close my PR as it's useless.